File: tests/fixtures/worker-pool/go/slow-go/handler.go
package main

import (
    "encoding/json"
    "time"
)

func handler(event map[string]interface{}) interface{} {
    time.Sleep(200 * time.Millisecond)
    body, _ := json.Marshal(map[string]interface{}{
        "runtime": "go",
        "ok": true,
    })
    return map[string]interface{}{
        "status": 200,
        "headers": map[string]string{"Content-Type": "application/json"},
        "body": string(body),
    }
}

File: examples/functions/python/slow/handler.py
import json
import time


def handler(event):
    query = event.get("query") or {}
    try:
        sleep_ms = int(query.get("sleep_ms") or 0)
    except Exception:
        sleep_ms = 0

    if sleep_ms > 0:
        time.sleep(sleep_ms / 1000.0)

    return {
        "status": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps(
            {
                "runtime": "python",
                "function": "slow",
                "slept_ms": sleep_ms,
            },
            separators=(",", ":"),
        ),
    }
